src/resistance.jl

```diff
@@ -0,0 +1,52 @@
+const HILL_COEFFICIENT = Param(0.0; bounds=(0.0, 100.0))
+const DEVIATION_PHENOTYPE = Param(0.0; bounds=(0.0, 100.0))
+const DOMINANCE_DEGREE = Param(0.0; bounds=(0.0, 100.0))
+
+"""
+Extends Resistance for using heterogenous data.
+"""
+abstract type Resistance{R,W} <: CellRule{R,W} end
+
+"""
+Lande Equation application
```

Comment:
Constructor signature at the top of the docs, more explanation of the equation (citation if possible), and list of the arguments as above.